web_widget_json_graph/static/src/js/web_widget_json_graph.js

```diff
@@ -0,0 +1,27 @@
+/* eslint no-undef: 0*/
```

Comment:
What error are you avoiding here?

Could it be specific to the problematic line?

Reply:
Done, moved to line 18